### PR TITLE
✏️ Add link to ReScript bindings

### LINKED ISSUE #360: Add Bucklescript bindings for Reasonml
fast-check is a very nice library. It would be even better if we could use it with Facebook's ReasonML instead of Typescript!

It might be possible to use https://github.com/rrdelaney/ReasonablyTyped to make the conversion easier.

### PR DESCRIPTION
**Pull request by TheSpyder**
Fixes #360 by including a link to my bindings

## In a nutshell

❌ New feature
❌ Fix an issue
✔️ Documentation improvement
❌ Other: *please explain*

(✔️: yes, ❌: no)

## Potential impacts

None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,12 @@ Here are the minimal requirements to use fast-check properly without any polyfil
 
 (1) Except for features that cannot be polyfilled - such as `bigint`-related ones - all the capabilities of fast-check should be usable given you use at least the minimal recommended version of node associated to your major of fast-check.
 
+## ReScript bindings
```
Comment (dubzzz) [inline README.md:113]: I'd rather put it directly into the Compatibility section with a bold label, something like
```suggestion
**ReScript bindings**

```

File: README.md [hunk 2]
```diff
@@ -110,6 +110,12 @@ Here are the minimal requirements to use fast-check properly without any polyfil
 
 (1) Except for features that cannot be polyfilled - such as `bigint`-related ones - all the capabilities of fast-check should be usable given you use at least the minimal recommended version of node associated to your major of fast-check.
 
+## ReScript bindings
+
+Bindings to use fast-check in the [ReScript](https://rescript-lang.org) langauge are available. These are maintained by [@TheSpyder](https://github.com/TheSpyder) external to the main project; please do not log binding-specific issues here.
+
+https://www.npmjs.com/rescript-fast-check
```
Comment (dubzzz) [inline README.md:117]:
```suggestion
Bindings to use fast-check in [ReScript](https://rescript-lang.org) are available in package [rescript-fast-check](https://www.npmjs.com/rescript-fast-check). They are maintained by [@TheSpyder](https://github.com/TheSpyder) as an external project.
```

**Review by dubzzz — COMMENTED** [review]
Some suggestions

File: README.md (hunk 1 above)
Comment (TheSpyder) [reply README.md:113]: Fair enough!

**Review by dubzzz — APPROVED** [review]

**Comment by dubzzz** [comment]
@TheSpyder Thanks a lot for the contribution and for the package!

**Pull request merged**